cmd/dockerRegistry.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var DOCKER_REGISTRY_NAME string = "docker-image-reg"
+
+func installDockerRegistry() {
```

Comment:
this file probably belongs in a different folder and not the command folder. cmd folder is specific to all cli entry points.